src/app/utils/BountyUtils.ts

```diff
@@ -63,27 +66,36 @@ const BountyUtils = {
 	},
 
 	async validateReward(guildMember: GuildMember, reward: BountyReward): Promise<void> {
-		const ALLOWED_CURRENCIES = ['BANK'];
+		const ALLOWED_CURRENCIES = ['BANK', 'ETH', 'BTC', 'USDC', 'USDT', 'TempCity'];
```

Comment:
It makes better sense to follow the approach outlined below in another comment regarding adding servers, so you're not having to hardcode currency as well. This way you can update the db collection with tokens and server IDs and not have to raise a PR every time you want to support a currency.